client/account-settings/components/label-payment-method/style.scss

```diff
@@ -0,0 +1,38 @@
+.card.is-compact.label-payment-method {
+	margin-bottom: 14px;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	cursor: pointer;
+
+	.payment-logo {
+		float: left;
+		margin-top: 0;
```

Comment:
It was already set `margin-top: 0;`, just in a different rule: https://github.com/Automattic/woocommerce-services/pull/1051/files/995d9174509fc081ffa6a5d130b4007088fedcaf#diff-a3c69445a234ef3dfa87f5b244ac40acL22